sormas-app/app/src/main/java/de/symeda/sormas/app/rest/SynchronizeDataAsync.java

```diff
@@ -178,6 +178,7 @@ protected Void doInBackground(Void... params) {
 				// Pull changed data and push existing data that has been changed on the mobile device
 				syncCallbacks.ifPresent(c -> c.getUpdateSynchronizationStepCallback().accept(SynchronizationDialog.SynchronizationStep.SYNCHRONIZE));
 				synchronizeChangedData();
+				pushNewPullMissingAndDeleteInvalidData();
```

Comment:
This is not an appropriate solution as this method should not be called when synchronizing only changes; it's meant to be used when doing a complete sync, e.g. after a previous sync has failed or when doing the initial sync after logging in. The logic here has to remain the same, i.e. only when it's been 24 hours since we last removed obsolete data, we do it by calling pullAndRemoveObsoleteUuidsSince.

Reply:
the ticket was changed and consists only of fixing the environment entities that are obsolete (archived/soft deleted) and that are not removed from the mobile app. The synchronization issue related to the jurisdiction change of core entities will be dealt with in a different ticket #12503.